app/src/main/java/com/github/geohunt/app/ui/components/activehunts/ActiveHunts.kt

```diff
@@ -100,11 +99,13 @@ fun TitleText() {
  * @param challenges the challenges to display
  */
 @Composable
-fun ActiveHuntsList(challenges: List<LazyRef<Challenge>>, emptyScreenCallback: () -> Unit) {
+fun ActiveHuntsList(challenges: List<LazyRef<Challenge>>, navController: NavController) {
     //wrapper Box
     Box(modifier = Modifier.fillMaxSize(), contentAlignment = Alignment.Center) {
         if(challenges.isEmpty()) {
-            EmptyChallengesScreen(emptyScreenCallback)
+            EmptyChallengesScreen() {
+                navController.explore()
+            }
```

Comment:
Avoid using trailing lambda's syntax for functions that do not represent `Composable`s, since it can be confused with them, and use named parameters instead. (Especially since `EmptyChallengesScreen` looks like a `Composable`, I assumed that the braces were meant for its content...)